SingularityUI/app/templates/requestForm.hbs

```diff
@@ -0,0 +1,172 @@
+    <h3>
+      {{#if model.id}}
+        Editing <a href="{{appRoot}}/request/{{model.id}}">{{model.id}}</a>
+      {{else}}
+        New Request
+      {{/if}}
+    </h3>
+    <form>
+      <div class="form-group {{#unless model.id}}required{{/unless}}">
+        {{#unless model.id}}
+        <label for="id">ID</label>
+        <input type="text" id="id" class="form-control" placeholder="eg: my-awesome-request">
+        {{/unless}}
+      </div>
+      <div class="form-group">
+        <label for="owner">Owners <span class='form-label-tip'>comma separate values</span></label>
+        <input id="owners" class="tagging-input">
+      </div>
+      <div class="form-group">
+        <label>Type</label>
+        <div id="type" class="btn-group">
+          <button data-type="SERVICE" class="btn btn-default">
+            Service
+          </button>
+          <button data-type="WORKER" class="btn btn-default">
+            Worker
+          </button>
+          <button data-type="SCHEDULED" class="btn btn-default">
+            Scheduled
+          </button>
+          <button data-type="ON_DEMAND" class="btn btn-default">
+            On-demand
+          </button>
+          <button data-type="RUN_ONCE" class="btn btn-default">
+            Run-once
+          </button>
+        </div>
+      </div>
+      <div class="form-group">
+        <label for="slavePlacement">Slave Placement</label>
+        <select id="slavePlacement" class="form-control">
+          <option value="">Default</option>
+          <option value="SEPARATE">Separate</option>
+          <option value="OPTIMISTIC">Optimistic</option>
+          <option value="GREEDY">Greedy</option>
+        </select>                
+      </div>
+
+      <div id="SERVICE-expandable" class="expandable hide">
+        <div class="row">
+          <div class="col-sm-4">
+            <div class="form-group">
+              <label for="instances-SERVICE" class="col-sm-6">Instances</label>
+              <div class="col-sm-6">
+                <input type="text" id="instances-SERVICE" class="form-control" placeholder="1">
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-4">
+            <div class="form-group">
+              <label for="rack-sensitive-SERVICE" data-tooltip='rack-sensitive'>
+                <input type="checkbox" id="rack-sensitive-SERVICE">
+                Rack sensitive
+              </label>
+            </div>
+          </div>
+          <div class="col-sm-4">
+            <div class="form-group">
+              <label for="load-balanced"{{#if model.id}} class="subtle"{{/if}}>
+                <input {{#if model.id}}disabled="disabled"{{/if}} type="checkbox" id="load-balanced">
+                Load balanced
+              </label>
+            </div>
+          </div>
+        </div>
+        <div class="form-group">
+          <label for="">Milliseconds to wait after task finishes for reschedule</label>
```

Comment:
it's actually foolish to allow people to do this on web services, so let's remove this field for Services